### DISCUSSION TITLE
Error 215: Cannot find precipitation maps

### DISCUSSION THREAD

**Discussion by weixiaoxun** [iiasa/CWatM#121] [Q&A] [answered]
do i need to download the precipitation map files from an FTP server? Thank you.

**Comment by mikhailsmilovic** [comment, marked as answer]
[Global observed climate, in 10-year chunks](https://drive.google.com/drive/folders/1_xlg_RZvx9xsVPddOiqCG3XZmASCBIRM?usp=drive_link) at 30 arcminute resolution are available on Google Drive 💧

**Reply by weixiaoxun** [reply]
thank you so much！

**Comment by mikhailsmilovic** [comment]
Guten Morgen,

The path to the climate data in the settings.ini file needs to be updated
to the path on your computer 🌤️
Now it's point to C:\Users\smilovic\OneDrive -
IIASA\cwatm_input\Climate\GSWP3-W5E5 does

Viele Grüße
Mikhail

**Reply by georasaq** [reply]
Hi! Pls am facing same problem error 215 "cannot find precipitation error" trying to update the climate data path on my computer "
![cwat error 215](https://github.com/user-attachments/assets/dab3e430-f36f-4c26-b7fe-e9afb667295c)
![cwatm error 215](https://github.com/user-attachments/assets/3112b42c-7dfb-4396-afe8-45b6ab160626)

"

**Reply by mikhailsmilovic** [reply]
Guten Morgen, the zipped climate folder needs to be unzipped 📂
